charts/fleet-agent/fleet-agent/100.0.1+up0.3.7/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+annotations:
```

Comment:
One thing we missed was to add the rancher-max to previous charts that are user facing:

* OPA gatekeeper - needs max, last version was 100.0.0+up3.5.1.tgz
* CIS - needs max since underlying image versions are changing
* Backup - needs max since underlying image versions are changing. Technically 2.0.0 is greater than any 1.x.0, but you could still "downgrade" any backports. This one and CIS are actually worse because its not obvious to user that its a downgrade.
* Istio - doesn't actually matter since its same underlying version and its just a bug fix. We'll have to max both in future, but now it doesn't matter. 

So 3 charts from 2.6.0 need max of < 2.6.1-0.

Reply:
This is being addressed in a follow-up PR by @jakefhyde